File: dms-frontend-users/src/app/model/add-user.ts
import {Attribute} from "./attribute";

export class AddUser {
  public username: string;
  public firstName: string;
  public lastName: string;
  public email: string;
  public realmRoles: Array<string>;
  public attributes: Attribute;


  constructor(username: string, firstName: string, lastName: string, email: string, realmRoles: Array<string>, attributes: Attribute) {
    this.username = username;
    this.firstName = firstName;
    this.lastName = lastName;
    this.email = email;
    this.realmRoles = realmRoles;
    this.attributes = attributes;
  }
}

File: dms-backend-users/src/main/java/com/dms/backend/users/model/AddUser.java
package com.dms.backend.users.model;

import lombok.Data;

import java.util.Arrays;

@Data
public class AddUser {
    String username;
    String firstName;
    String lastName;
    String email;
    String[] realmRoles;
    Attribute attributes;

    @Override
    public String toString() {
        return "AddUser{" +
                "username='" + username + '\'' +
                ", firstName='" + firstName + '\'' +
                ", lastName='" + lastName + '\'' +
                ", email='" + email + '\'' +
                ", realmRoles=" + Arrays.toString(realmRoles) +
                ", attributes=" + attributes +
                '}';
    }
}
